types/error/GeneralError.raml

```diff
@@ -0,0 +1,5 @@
+#%RAML 1.0 DataType
+(package): Error
+type: ErrorObject
+displayName: GeneralError
+discriminatorValue: General
```

Comment:
not sure if it's correct to add this error but it's in the documentation

Reply:
it's a valid error